Plugins/BTCPayServer.Plugins.ShopifyPlugin/Views/UIShopify/Settings.cshtml

```diff
@@ -10,9 +10,11 @@
     var storeId = (string)this.Context.GetRouteValue("storeId");
     var pluginUrl = this.Url.ActionAbsolute(Context.Request, nameof(Index), new { storeId = storeId }).AbsoluteUri;
 
+    bool allStepsCompleted = Model.Step == ShopifySettingsViewModel.State.Done;
     bool expandClientApp = Model.Step == ShopifySettingsViewModel.State.WaitingClientCreds;
     bool expandDeploy = Model.Step == ShopifySettingsViewModel.State.WaitingForDeploy;
-    bool expandInstall = Model.Step == ShopifySettingsViewModel.State.WaitingForInstall || Model.Step == ShopifySettingsViewModel.State.Done;
+    bool expandInstall = !allStepsCompleted && Model.Step == ShopifySettingsViewModel.State.WaitingForInstall;
```

Comment:
So the reason I wanted to expand install, even if it is completed is that you want the user to easily see the next step for him to do.

But since we can't verify on our side if it has been done, we can just show the information.

```
You installed this app on your Shopify store. Do not forget to <a href="@Model.ShopCheckoutSettingsUrl" rel="noreferrer noopener" target="_blank">customize your store</a> by adding the UI Extension to the "Thank you" page.
```

Reply:
Yes.. this is updated on the admin view: https://github.com/btcpayserver/btcpayserver-shopify-plugin/pull/1/commits/51baf6ad3fb7e46077121d82009a913a3366bf1a